### PR TITLE
 UX Stuff: modifing search input,removing category section shadow

### PR DESCRIPTION
**Pull request by binarydevteam**
UX Stuff: modifing search input,removing category section shadow,underlying assets subcategory,remove background highlight of category filter
@brucebinary

### CODE REVIEW

File: sass/components/_categorical-display.scss [hunk 1]
```diff
@@ -75,7 +94,6 @@
         box-sizing: border-box;
         transition: background-color 0.3s;
         &:hover {
```
Comment (brucebinary) [inline sass/components/_categorical-display.scss]: just remove hover altogether; don't leave it blank

File: sass/components/_categorical-display.scss [hunk 2]
```diff
@@ -16,25 +16,44 @@
...
             border: none;
             outline: 0;
-            width: calc(100% - 30px);
-            border-bottom: 1px solid #f0f0f0;
+            width: calc(100% - 40px);
+            border-radius: 4px;
             @include placeholder {
                 color: #dfdfdf;
             }
             &:focus {
```
Comment (brucebinary) [inline sass/components/_categorical-display.scss]: just remove hover altogether; don't leave it blank

File: src/store/CategoricalDisplayStore.js [hunk 3]
```diff
@@ -197,6 +197,15 @@ export default class CategoricalDisplayStore {
         }, 0);
     }
 
+    @action.bound clearFilterText() {
+        this.filterText = '';
+        this.searchInput.value = '';
+        setTimeout(() => {
+            this.scroll.refresh();
```
Comment (brucebinary) [inline src/store/CategoricalDisplayStore.js]: `this.scroll.refresh()` isn't used anymore now that we replace iScroll with perfect-scrollbar. Did you update your branch?

File: src/store/CategoricalDisplayStore.js [hunk 4]
```diff
@@ -197,6 +197,15 @@ export default class CategoricalDisplayStore {
         }, 0);
     }
 
+    @action.bound clearFilterText() {
+        this.filterText = '';
+        this.searchInput.value = '';
```
Comment (brucebinary) [inline src/store/CategoricalDisplayStore.js]: I think either only `this.filterText = '';` or `this.searchInput.value = '';` is needed. Check back with me and see?

File: src/store/CategoricalDisplayStore.js (hunk 3 above)
Comment (binarydevteam) [reply src/store/CategoricalDisplayStore.js]: so I should just use `this.updateScrollSpy()` ??

File: src/store/CategoricalDisplayStore.js (hunk 4 above)
Comment (binarydevteam) [reply src/store/CategoricalDisplayStore.js]: one of them responsible to clear input text, and another one to reset filter. if we remove any of them, the search and it's input do not work proper.

File: src/store/CategoricalDisplayStore.js (hunk 3 above)
Comment (brucebinary) [reply src/store/CategoricalDisplayStore.js]: @binarydevteam I think you should. That updates which category is highlighted. If you remove that line, chances are when you clear the filter text the active category will be in the wrong place. 

When in doubt just remove and see if it still works

File: src/store/CategoricalDisplayStore.js (hunk 4 above)
Comment (brucebinary) [reply src/store/CategoricalDisplayStore.js]: aite gotcha

File: src/store/CategoricalDisplayStore.js [hunk 5]
```diff
@@ -198,12 +198,8 @@ export default class CategoricalDisplayStore {
     }
 
     @action.bound clearFilterText() {
-        this.filterText = '';
+        this.setFilterText('');
         this.searchInput.value = '';
-        setTimeout(() => {
-            this.scroll.refresh();
-            this.updateScrollSpy();
-        }, 0);
```
Comment (brucebinary) [inline src/store/CategoricalDisplayStore.js]: does the active category update when you remove this? Did you check?

Comment (binarydevteam) [reply src/store/CategoricalDisplayStore.js]: @brucebinary  yeah, it get update when I click `times icon` or `clear search text`

Comment (binarydevteam) [reply src/store/CategoricalDisplayStore.js]: @brucebinary  check my last changes, I double check with them, and all work fine

**Pull request merged**